diff --git a/node_modules/react-native-health-connect/android/build.gradle b/node_modules/react-native-health-connect/android/build.gradle
index 5b5b4dc..ed28a38 100644
--- a/node_modules/react-native-health-connect/android/build.gradle
+++ b/node_modules/react-native-health-connect/android/build.gradle
@@ -92,7 +92,7 @@ dependencies {
   implementation "com.facebook.react:react-native"
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
   implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.7.3'
-  implementation "androidx.health.connect:connect-client:1.1.0-alpha11"
+  implementation "androidx.health.connect:connect-client:1.1.0-alpha12"
 }

 if (isNewArchitectureEnabled()) {
